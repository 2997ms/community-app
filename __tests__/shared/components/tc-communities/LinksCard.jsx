import React from 'react';
import Rnd from 'react-test-renderer/shallow';
import LinksCard from 'components/tc-communities/LinksCard';

const rnd = new Rnd();

test('Snapshot match', () => {
  rnd.render((
    <LinksCard
      title="Videos"
      links={[{
        title: 'Tristique ullamcorper id vitae',
        url: '#',
      }, {
        title: 'Nulla mollis sapien sollicitudin',
        url: '#',
      }, {
        title: 'Vivamus facilisis dolor et massa',
        url: '#',
      }, {
        title: 'Vestibulum nisl egestas',
        url: '#',
      }]}
    />
  ));
  expect(rnd.getRenderOutput()).toMatchSnapshot();

  rnd.render((
    <LinksCard
      title="Videos"
      links={[{
        title: 'Tristique ullamcorper id vitae',
        url: '#',
      }, {
        title: 'Nulla mollis sapien sollicitudin',
        url: '#',
      }, {
        title: 'Vivamus facilisis dolor et massa',
        url: '#',
      }, {
        title: 'Vestibulum nisl egestas',
        url: '#',
      }]}
      theme={{
        container: 'container',
        title: 'title',
        list: 'list',
        item: 'item',
        link: 'link',
      }}
    />
  ));
  expect(rnd.getRenderOutput()).toMatchSnapshot();
});
