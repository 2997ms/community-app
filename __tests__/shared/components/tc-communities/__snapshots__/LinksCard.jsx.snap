// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot match 1`] = `
<LinksCard
  links={
    Array [
      Object {
        "title": "Tristique ullamcorper id vitae",
        "url": "#",
      },
      Object {
        "title": "Nulla mollis sapien sollicitudin",
        "url": "#",
      },
      Object {
        "title": "Vivamus facilisis dolor et massa",
        "url": "#",
      },
      Object {
        "title": "Vestibulum nisl egestas",
        "url": "#",
      },
    ]
  }
  theme={
    Object {
      "container": "src-shared-components-tc-communities-LinksCard-___style__container___1im8e",
      "item": "src-shared-components-tc-communities-LinksCard-___style__item___2Z3tF",
      "link": "src-shared-components-tc-communities-LinksCard-___style__link___AMsX5",
      "list": "src-shared-components-tc-communities-LinksCard-___style__list___1f_KC",
      "title": "src-shared-components-tc-communities-LinksCard-___style__title___wooM4",
    }
  }
  title="Videos"
/>
`;

exports[`Snapshot match 2`] = `
<LinksCard
  links={
    Array [
      Object {
        "title": "Tristique ullamcorper id vitae",
        "url": "#",
      },
      Object {
        "title": "Nulla mollis sapien sollicitudin",
        "url": "#",
      },
      Object {
        "title": "Vivamus facilisis dolor et massa",
        "url": "#",
      },
      Object {
        "title": "Vestibulum nisl egestas",
        "url": "#",
      },
    ]
  }
  theme={
    Object {
      "container": "src-shared-components-tc-communities-LinksCard-___style__container___1im8e container",
      "item": "src-shared-components-tc-communities-LinksCard-___style__item___2Z3tF item",
      "link": "src-shared-components-tc-communities-LinksCard-___style__link___AMsX5 link",
      "list": "src-shared-components-tc-communities-LinksCard-___style__list___1f_KC list",
      "title": "src-shared-components-tc-communities-LinksCard-___style__title___wooM4 title",
    }
  }
  title="Videos"
/>
`;
